updating/preparing_for_updates/updating-cluster-prepare-past-4.18.adoc

```diff
@@ -0,0 +1,123 @@
+    beta.kubernetes.io/instance-type: m4.large
+    kubernetes.io/hostname: ip-10-0-131-14
+    beta.kubernetes.io/arch: amd64
+#...
+----
+<1> The label identifying the operating system that runs on the node, to match the pod node selector.
+
+Any pods that you want to schedule to new {op-system} nodes must contain a matching label in its `nodeSelector` field. The following procedure describes how to add the label.
+
+.Procedure
+
+. Deschedule the {op-system-base} node currently running your existing pods by entering the following command:
++
+[source,terminal]
+----
+$ oc adm cordon <rhel-node>
+----
+
+. Add an `rhcos` node selector to a pod:
+
+** To add the node selector to existing and future pods, add the node selector to the controller object for the pods by entering the following command:
++
+.Example `Deployment` object with `rhcos` label
+[source,terminal]
+----
+$ oc patch dc <my-app> -p '{"spec":{"template":{"spec":{"nodeSelector":{"node.openshift.io/os_id":"rhcos"}}}}}'
+----
++
+Any existing pods under your `Deployment` controlling object will be re-created on your {op-system} nodes.
+
+** To add the node selector to a specific, new pod, add the selector to the `Pod` object directly:
++
+.Example `Pod` object with `rhcos` label
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: <my-app>
+#...
+spec:
+  nodeSelector:
+    node.openshift.io/os_id: rhcos
+#...
+----
++
+The new pod will be created on {op-system} nodes, assuming the pod also has a controlling object.
+
+[id="identifying-and-removing-rhel-worker-nodes_{context}"]
+== Identifying and removing {op-system-base} worker nodes
+
+With the introduction of {product-title} 4.19, package-based {op-system-base} worker nodes are no longer supported. The following procedure describes how to identify {op-system-base} nodes for cluster removal on bare-metal installations. You must complete the following steps to successfully update your cluster.
+
+.Procedure
+
+. Identify nodes in your cluster that are running {op-system-base} by entering the following command:
++
+[source,terminal]
+----
+$ oc get -l node.openshift.io/os_id=rhel
+----
++
+.Example output
++
+[source,text]
+----
+NAME                        STATUS    ROLES     AGE       VERSION
+rhel-node1.example.com      Ready     worker    7h        v1.31.7
+rhel-node2.example.com      Ready     worker    7h        v1.31.7
+rhel-node3.example.com      Ready     worker    7h        v1.31.7
+----
+
+. xref:../../nodes/nodes/nodes-nodes-working.adoc#nodes-nodes-working-deleting-bare-metal_nodes-nodes-working[Continue with the node removal process]. {op-system-base} nodes are not managed by the Machine API and have no compute machine sets associated with them. You must unschedule and drain the node before you manually delete it from the cluster.
+
+For more information on this process, see link:https://access.redhat.com/solutions/4976801[How to remove a worker node from Red{nbsp}Hat {product-title} 4 UPI?].
```

Comment:
```suggestion
For more information on this process, see link:https://access.redhat.com/solutions/4976801[How to remove a worker node from Red{nbsp}Hat {product-title} 4 UPI?]
```
In the preview, it looked a little odd to have a period right after the question mark. I know the question mark is technically part of the title, but I think it might make sense to remove the period. WDYT?

Reply:
i agree with you, it looks strange. i'm down with omitting the question mark 😄